lib/rules/no-invalid-aria-attributes.js

```diff
@@ -0,0 +1,154 @@
+import { aria } from 'aria-query';
+
+import Rule from './_base.js';
+
+const VALID_ARIA_ATTRIBUTES = aria;
+
+function createInvalidAriaAttributeMessage(name) {
+  return `${name} is an unrecognized ARIA attribute.`;
+}
+
+function createInvalidAttributeTypeErrorMessage(name, type, permittedValues) {
+  switch (type) {
+    case 'tristate':
+      return `The value for ${name} must be a boolean or the string "mixed".`;
+    case 'token':
+      return `The value for ${name} must be a single token from the following: ${permittedValues.join(
+        ', '
+      )}.`;
+    case 'tokenlist':
+      return `The value for ${name} must be a list of one or more tokens from the following: ${permittedValues.join(
+        ', '
+      )}.`;
+    case 'idlist':
+      return `The value for ${name} must be a list of strings that represent DOM element IDs (idlist)`;
+    case 'id':
+      return `The value for ${name} must be a string that represents a DOM element ID`;
+    case 'integer':
+      return `The value for ${name} must be an integer.`;
+    default:
+      return `The value for ${name} must be a ${type}.`;
+  }
+}
+
+function isBoolean(value) {
+  return typeof value === 'boolean' || value === 'true' || value === 'false';
+}
+
+function isNumeric(value) {
+  if (typeof value === 'string') {
+    value = Number.parseInt(value, 10);
```

Comment:
Normally it's good to avoid modifying the input parameters. Maybe just use `Number.isInteger()`? Not sure why this is checking for booleans.